assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/LegacyDashboardAllTraffic.js

```diff
@@ -43,7 +44,7 @@ function LegacyDashboardAllTraffic() {
 			</Cell>
 			<Cell size={ 12 }>
 				<Layout className="googlesitekit-pagespeed-widget">
-					<DashboardAllTrafficWidget { ...widgetComponentProps } />
+					<DashboardAllTrafficWidget { ...widgetComponentProps } WidgetReportError={ ReportError } />
```

Comment:
@johnPhillips i think we don't need to pass it manually here since the `WidgetReportError` property is already passed when we destruct the `widgetComponentProps` object.

Reply:
@eugene-manuilov Maybe I misunderstood the IB here?

> Once #2429 is merged into develop, update the LegacyDashboardAllTraffic component to pass the non-scoped ReportError element to the DashboardAllTrafficWidget element as a fallback for the scoped version.

I interpreted that as needing to override the one from `widgetComponentProps` with an unscoped `ReportError`. Let me know if I've misunderstood?

Reply:
I think it is my mistake in the IB. Can you check whether the `widgetComponentReport` has `WidgetReportError` property? If it has it, then we don't need to override it.

Reply:
I couldn't find this, but perhaps I am misunderstanding. I've removed the prop here 👍